releases/v20.2.7.md

```diff
@@ -0,0 +1,93 @@
+---
+title: What&#39;s New in v20.2.7
+toc: true
+summary: Additions and changes in CockroachDB version v20.2.7 since version v20.2.6
+---
+
+## March 24, 2021
+
+This page lists additions and changes in v20.2.7 since version v20.2.6.
+
+- For a comprehensive summary of features in v20.2, see the [v20.2 GA release notes](v20.2.0.html).
+- To upgrade to v20.2, see [Upgrade to CockroachDB v20.2](../v20.2/upgrade-cockroach-version.html).
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v20.2.7
+~~~
+
+
+### SQL language changes
+
+- The following [cluster settings](cluster-settings.html) were added: 
+  - `sql.defaults.statement_timeout` controls the default value for the statement_timeout session setting.
+  - `sql.defaults.idle_in_transaction_session_timeout` controls the default value for the `idle_in_transaction_session_timeout` timeout setting.
+  - `sql.defaults.idle_in_session_timeout` already existed, but is now a public cluter setting. [#62185][#62185]
+
+### Bug fixes
+
+- Previously, [changefeeds](change-data-capture.html) were checking the `SELECT` privilege on every descriptor associated with targeted tables, including the database and custom types. They now only check `SELECT` on the targeted tables. [#61661][#61661]
+- The names of custom types are no longer sent to Cockroach Labs in telemetry and crash reports. [#61645][#61645]
+- Fixed a bug where some import failures would cause tables to stay `OFFLINE`, when they should have been brought back to `PUBLIC`. [#61480][#61480]
```

Comment:
```suggestion
- Fixed a bug where some import failures would cause tables to stay `OFFLINE` when they should have been brought back to `PUBLIC`. [#61480][#61480]
```